.travis.yml

```diff
@@ -1,26 +1,18 @@
 language: go
 
 go:
-  - 1.11.x
+  - 1.12.x
 
 env:
-  - DEP_VERSION="0.5.0"
+  - GO111MODULE=on
```

Comment:
do we need this provided that Go version is 1.12 now?

Reply:
it is needed for travis, it will be `on` by default from 1.13 AFAIK